python/pyspark/sql/column.py

```diff
@@ -303,6 +303,25 @@ def getItem(self, key):
         """
         return _bin_op("getItem")(self, key)
 
+    @ignore_unicode_prefix
+    @since(3.0)
```

Comment:
It will be 3.1 since it's too late for 3.0. Can we remove the python part in this PR, and focus on merging the core Scala part?